plugins/webhooks/index.js

```diff
@@ -232,6 +237,21 @@ async function createPREvents(options, request) {
     return Promise.all(events);
 }
 
+/**
+ * Resolve ChainPR flag
+ * @method resolveChainPR
+ * @param  {Boolean}  chainPR              Plugin Chain PR flag
+ * @param  {Pipeline} pipeline             Pipeline
+ * @param  {Object}   pipeline.annotations Pipeline-level annotations
+ * @return {Boolean}
+ */
+function resolveChainPR(chainPR, pipeline) {
+    const defaultChainPR = typeof chainPR === 'undefined' ? false : chainPR;
+    const annotChainPR = pipeline.annotations[ANNOT_CHAIN_PR];
+
+    return typeof annotChainPR === 'undefined' ? defaultChainPR : annotChainPR;
```

Comment:
Does this work here?
```suggestion
    return annotChainPR || defaultChainPR;
```

Reply:
No. If `annotChainPR` is false and `defaultChainPR` is true, it returns true, but we expect false.

Reply:
Ah thanks for explaining, makes sense